releasenotes/notes/upgrade-pulse-builder-and-rzx-builder-033ac8ad8ad2a192.yaml

```diff
@@ -0,0 +1,26 @@
+---
+features:
+  - |
+    New pulse directive :class:`~qiskit.pulse.instructions.AreaBarrier` has been added.
+    This instruction is QPY compatible. This directive behaves almost identically to
+    delay instruction, but will be removed before execution.
+    This directive is intended to be used internally within the pulse builder
+    and helps :class:`.ScheduleBlock` with representing instructions with
```

Comment:
```suggestion
    and helps :class:`.ScheduleBlock` represent instructions with
```

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/8949/commits/0ca8efbc78481b5edd7456ec02b026a96785981a